docs/vtex-io/Perks/best-practices/vtex-io-documentation-best-practices-for-optimizing-performance.md

```diff
@@ -6,171 +6,171 @@ createdAt: "2020-11-11T12:55:53.602Z"
+> ⚠️ Keep in mind that by enabling this option, you might notice some style inconsistencies in the store layout. Test it with caution before enabling it on a production workspace. Also, if you notice any side-effects on the store website, please [open a ticket](https://help-tickets.vtex.com/smartlink/sso/login/zendesk) and let us know!
 
 ### Improving download speed with CSS concatenation
 
-To properly display a web page layout, the browser needs to make multiple HTTP requests to download each CSS file. However, numerous requests might increase the time taken to render a single web page.
+To correctly display a web page layout, the browser needs to make multiple HTTP requests to download each CSS file. However, numerous requests might increase the time to render a single web page.
 
-In this sense, VTEX IO offers the possibility of avoiding various download requests by concatenating multiple CSS files into one.
+VTEX IO can avoid various download requests by concatenating multiple CSS files in one.
 
 ![concatenation](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@main/images/vtex-io-documentation-best-practices-for-optimizing-performance-2.png)
 
-Notice that, by combining CSS in a single file, the browser makes fewer requests and loads web pages faster, improving your store website's performance.
+Note that by combining CSS in a single file, the browser makes fewer requests and loads web pages faster, improving the performance of your ecommerce website.
 
 ### Lazy loading page metadata
 
-Before displaying a web page, the VTEX IO service app responsible for server-side rendering, or simply, the Render Runtime, interprets the page's metadata script. An example of the before-mentioned script is presented below.
+Before displaying a web page, the VTEX IO service app for server-side rendering, or simply Render Runtime, interprets the page metadata script. You can see an example of this script below.
 
 ```ts
 <script>
         __RUNTIME__ = {"account":"vtexstore","amp":false,"bindingChanged":false,"binding":{"id":"aacb04t3-a8fa-4bab-b5bd-2d654d20dcd8","canonicalBaseAddress":"vtexstore.vtex.com"},"culture":{"availableLocales":[],"country":"USA","currency":"USD","language":"en","locale":"en-US","customCurrencyDecimalDigits":null,"customCurrencySymbol":"$"},"production":true,"query":{},"settings":{....
 ```
 
-Notice that this script holds many important data about a web page and it can be long, characterizing a long task to be performed by the browser.
+Note that this script holds a lot of important data about a web page, and it can be long, which can be a big task for the browser.
 
-Therefore, to avoid your store website's total blocking time to be compromised, by enabling the lazy runtime, such script will be broken into smaller ones.
+Therefore, to prevent the total blocking time of the store website from being compromised, this script will be broken into smaller ones when lazy runtime is enabled.
```

Comment:
```suggestion
Therefore, when lazy runtime is enabled, this script is broken into smaller ones in order to prevent the total blocking time of the store website from being compromised.
```